src/advanced/node-management.md

```diff
@@ -559,258 +556,193 @@ Response is a JSON object with the following fields:
-          "service_id": 0,
-          "signature": "e6cb64f5d7a80e10fd8367a8379342c32ecb164906002ada775643000f4cbde7e1735a068c17aa7de0ce491fdfa05b3de9519018e4370e13f48675857d4e0307"
-        },
-        {
-          "body": {
-            "block_hash": "272913062630e0e6ec3f7db1db91811052b829f0fdf1f27a0aec212f1684cf76",
-            "height": "20",
-            "propose_hash": "8fcca116a080ccb0d2b31768f7c03408707d595ec9b48813a2e8aef2b95673cd",
-            "round": 2,
-            "time": "2018-05-17T10:49:11.161549Z",
-            "validator": 0
-          },
-          "message_id": 4,
-          "protocol_version": 0,
-          "service_id": 0,
-          "signature": "2eab829b3fc123025df6adac3f06bbafcd7882cee7601ad7791e5cc1171349c9f107b229543ee89cff2c323aafef228e850da36c4578c6c593fbb085a079d60e"
-        }
-      ],
-      "txs": []
+      "precommits": ["a410964c2c21199b48e278b64bb72e2b8b20374df1ba5c8a846d34de9254a706010008031001180222220a2017ba87030093c7f27a73d7f987f36d0b731c015a1fdefe9d2799e45eaa26148f2a220a202e2c737dc5e902084b252991dbcd7c978565bb76b271d27681a675c81bdbbfae320b08e4ee95e30510f09da312b91820e1a9b0132c32d608e5206bf5fe119c54c17424ef6f6c70b13490761ddd36f855e2c74e37c7aa7e1ac648893164a07ed413c2c0065738d6bea8825bbf04", "0776a07b194e1a9b918205331e0c1f62de82d5b23efccb5922624cb928b620c901001001180222220a2017ba87030093c7f27a73d7f987f36d0b731c015a1fdefe9d2799e45eaa26148f2a220a202e2c737dc5e902084b252991dbcd7c978565bb76b271d27681a675c81bdbbfae320b08e4ee95e30510e8bfe11105fcd4562131756c6f6c64d0e63f4cca67bf718fb43a1ca45161ee3328c97c2582ab70f40406d4b2aa5fc967e2b177b897a3cf2dc0a674df4eccb45f75db8900", "57b745d4e157299ede29129ba039a039e6f145aae3852481937ac1972b2ed131010008011001180222220a2017ba87030093c7f27a73d7f987f36d0b731c015a1fdefe9d2799e45eaa26148f2a220a202e2c737dc5e902084b252991dbcd7c978565bb76b271d27681a675c81bdbbfae320b08e4ee95e30510d8eee11143e792e3c8a72403fc3dc259f3e8e0c3de867be4555fb809e7c8f79ce9449d15020b9060eb6d41efb5079bd25147dc2a5f3071b9bb7ed2fc8751468e750b310d"],
+      "txs": ["336a4acbe2ff0dd18989316f4bc8d17a4bfe79985424fe483c45e8ac92963d13"],
+      "time": "2019-02-14T14:12:52.037255Z"
     }
     ```
+<!-- markdownlint-enable MD013 -->
 
-### Blocks in range
+### Blocks in Range
 
 ```none
-GET {explorer_base_path}/blocks?count={count}&skip_empty_blocks={skip}&latest={latest}
+GET {explorer_base_path}/blocks?count={count}&skip_empty_blocks={skip}&latest={latest}&add_blocks_time={add}
 ```
 
-Returns the explored range and the corresponding headers. The range specifies
-the smallest and largest heights traversed to collect at most `count` blocks.
+Returns the block headers from the specified range. The range
+defines its smallest and largest block heights. The amount of collected
+blocks from the traversed range should not exceed `count` value.
 
 #### Parameters
 
 - **count**: integer  
-  The number of blocks to return. Should not be greater than
+  Number of blocks to return. Should not be greater than.
```

Comment:
The dot should be at the end of the sentence:
Number of blocks to return. Should not be greater than
  [`MAX_BLOCKS_PER_REQUEST`][github_explorer].